src/ostree/ot-admin-builtin-nsenter.c

```diff
@@ -0,0 +1,126 @@
+/*
+* Copyright (C) 2024 Colin Walters <walters@verbum.org>
```

Comment:
I think this is actually Copyright to you

Reply:
Hmm... If we consider it seriously, then every file I have modified needs to add my name. I don't want that. I think maybe a better approach is to write "Copyright (C) 2024 Colin Walters and contributors"

Reply:
Actually on second thought I'd personally vote to just drop `Copyright` and `Author`. Both linux and systemd for example seem to have just the SPDX license identifier in new files and that's it.

Reply:
We can do that in another PR. Let's keep this for now.